src/lib/features/project/project-store.ts

```diff
@@ -742,15 +742,17 @@ class ProjectStore implements IProjectStore {
                     instances: [row.instance_id],
                     sdks: [getSdk(sdkParts)],
                 };
-                entriesMap.set(row.feature_name, entry);
+                entriesMap.set(row.app_name, entry);
```

Comment:
This change seems significant. I assume it's intended.

Reply:
Yes, this was typo basically, which was caught by test.